packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -1489,14 +1490,17 @@ export async function postApiKeyReveal(
 }
 
 export async function getWebhooks(req: AuthRequest, res: Response) {
-  const { org } = getContextFromReq(req);
+  const context = getContextFromReq(req);
   const webhooks = await WebhookModel.find({
```

Comment:
I've created a follow up [ticket](https://github.com/growthbook/growthbook/issues/2086) to refactor the Webhook model so we're not exporting the model.